### ISSUE TITLE
Cannot access the Title, Author or Date fields

### ISSUE THREAD

**Issue by david-iterators** [CityOfBoston/Iterators#111]
Screengrab/videos (insert file names here)
2020 02 24, Cannot access the Title, Author or Date fields
[2020 03 24 Cannot access Title Author or Date fields.mp4.zip](https://github.com/CityOfBoston/Iterators/files/4378215/2020.03.24.Cannot.access.Title.Author.or.Date.fields.mp4.zip)
.mp4

Link
https://d8-stg.boston.gov/news

Node # (if known)
1401

Type of User:  Anonymous

Environment
(Device > OS > Browser)
MacOS 10.15.3

Trusted Tester Test
4) Keyboard and Focus

Brief Description
When trying to tab through the website, node 1401, cannot access the title, author or date field while using the keyboard.

Steps to reproduce
1, 00:00, Go the https://d8-stg.boston.gov/news
2, 00:11, Using the keyboard only, tabbing through the page.
3. 00:18, Tabbed successfully to the “we also created an RSS feed for City of Boston news”
4. 00:20, Tabbing again lands on the “Apply” button for the Title, Author, Date field query. 
	This is an accessibility bug because you cannot land on the Title, Author or Date field using only the keyboard. 
5. 00:21, Tabbed again and landed on the link, Joint Statement from Mayor Walsh, which is correct. 
6. 00:23, Used Shift Tab feature to go and landed on the “Apply” button. 
7, 00:34, Used Shirt Tab feature again and landed on “we also created an RSS feed for City of Boston news” again.

Desired Result
•	Able to successfully navigate Node 1401 using only the keyboard for access.

Actual Result
•	Able to successfully navigate Node 1401 using only the keyboard for access.

**Comment by andrebtlima** [comment]
@subaha-cob You can now access the title, author, and date fields but when you tab to the author field and expand the departments checkbox dropdown, you cannot navigate through the checkboxes using the keyboard.

**Comment by subaha-cob** [comment]
Fixed the issue. Going through travis now. Fix should be in by the morning. Please test in UAT in the morning.

**Comment by david-iterators** [comment]
Verified on https://d8-uat.boston.gov/news
